data-sources/flashdb/config.yml

```
@@ -0,0 +1,29 @@
+id: FlashDB
```

This should be lowercase
```suggestion
id: flashdb
```